CMSgov/easi-app
# [EASI-1781] System intake document upload

# EASI-1781

[Figma
](https://www.figma.com/file/JE39JNLbCSha4fk1m9ayPP/IT-Governance-Updates?node-id=311%3A4286&t=3397hHiAtYSLJ40m-1)
## Changes and Description

- Adds document upload step to system intake form
- Displays documents on review step of intake form
- updated unit and Cypress tests



## How to test this change

1. Complete documents step of system intake form
2. Test document upload form
3. Test removing documents
5. Confirm that documents appear in review step of intake form

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
